classes/class-p4-settings.php

```diff
@@ -145,6 +145,13 @@ public function __construct() {
 					],
 				],
 
+				[
+					'name' => __( 'Donate button visible on mobile', 'planet4-master-theme-backend' ),
+					'desc' => __( 'Enable visibility of the donate button on the homepage. Please check legal restrictions on images associated with this button before using it.', 'planet4-master-theme-backend' ) . ' <a href="">' . __( 'Read more', 'planet4-master-theme-backend' ) . '</a>',
```

Comment:
The handbook page is not yet ready. once available the link will be update here.

Reply:
✅ done